hippunfold/workflow/scripts/laplace_coords_withinit.py

```diff
@@ -15,9 +14,14 @@
 
 lbl_nib = nib.load(snakemake.input.lbl)
 lbl = lbl_nib.get_fdata()
+print(f'loaded lbl', file=logfile, flush=True)
 
-init_coords_nib = nib.load(snakemake.input.init_coords)
+try:
```

Comment:
not sure why this is needed? 
(I haven't run it myself to investigate, but shouldn't init_coords always be a string, and not a list of strings?)

Reply:
right? so weird. It seems to come up sometimes for me. Maybe I just have something funny in my virtual environment but, either way, this should be safer